pallets/chocolate/Cargo.toml

```diff
@@ -12,7 +12,7 @@ readme = 'README.md'
 
 [package.metadata.docs.rs]
 targets = ['x86_64-unknown-linux-gnu']
-
+# remove -dev???
```

Comment:
Might be move the serde from dev-dependencies to dependencies?

Reply:
Cool, will move on the next commit